fork-visualizer/tailwind.config.js

```diff
@@ -0,0 +1,18 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+    content: [
+      './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
+      './src/components/**/*.{js,ts,jsx,tsx,mdx}',
+      './src/app/**/*.{js,ts,jsx,tsx,mdx}',
```

Comment:
```suggestion
      './src/pages/**/*.{js,jsx}',
      './src/components/**/*.{js,jsx}',
      './src/app/**/*.{js,jsx}',
```

Reply:
TIL https://mdxjs.com/docs/what-is-mdx/ "Markdown for the component era"

Reply:
I am using it heavily in the [Alumni site](https://github.com/Polkadot-Blockchain-Academy/alumni-hub/blob/main/src/pages/Ecosystem/index.mdx)